returnAddenda.go

```diff
@@ -0,0 +1,71 @@
+package ach
+
+// written
+
+import (
+	"flag"
+	"strings"
+	"time"
+)
+
+const (
+	timeFormat = "060102" // for date of death
```

Comment:
Remove this is already supported in converters.go->parseSimpleDate() and composed into returnAddenda.go